protoc-gen-gogo/protoc-gen-combo/combo.go

```diff
@@ -70,8 +73,22 @@ func (this MixMatch) Gen(folder string, news []string) {
 	var regenerate = exec.Command("protoc", args...)
 	out, err := regenerate.CombinedOutput()
 	if err != nil {
-		fmt.Printf("%s\n", string(out))
-		panic(err)
+		fmt.Print("protoc-gen-combo: error:", err)
```

Comment:
Could you maybe move the scanner that prints out the text to above the error check.
That way error or no error the output gets printed out and if there is any type of error os.Exit(1) is called.